apps/back/src/Model/User.php

```diff
@@ -3,21 +3,18 @@
 namespace App\Model;
 
 class User {
-    private string $name;
-    private string $surname;
-    private string $mail;
-    private $expenses = [];
+    private array $expenses;
```

Comment:
```suggestion
    private array $expenses = [];
```

Soit tu as oublié d'initialiser ton tableau ici, ou bien dans le constructeur. Mais sachant que tu l'as déclaré en temps que `array` et non `null|array` tu devrais le déclarer l'initialiser 👍